From: Gerald Schaefer <geraldsc@de.ibm.com>
Subject: IBM Codestream linux-2.6.16 october2005, patch base-07

[patch] kernel: connector support.

Include connector config in the s390 arch Kconfig to get support
for connectors.
Acked-by: Hannes Reinecke <hare@suse.de>

---

 arch/s390/Kconfig   |    2 ++
 arch/s390/defconfig |    5 +++++
 2 files changed, 7 insertions(+)

diff -urpN linux-2.6/arch/s390/defconfig linux-2.6-patched/arch/s390/defconfig
--- linux-2.6/arch/s390/defconfig	2006-03-22 11:46:40.000000000 +0100
+++ linux-2.6-patched/arch/s390/defconfig	2006-03-22 11:46:46.000000000 +0100
@@ -277,6 +277,11 @@ CONFIG_PREVENT_FIRMWARE_BUILD=y
 # CONFIG_DEBUG_DRIVER is not set
 
 #
+# Connector - unified userspace <-> kernelspace linker
+#
+CONFIG_CONNECTOR=m
+
+#
 # SCSI device support
 #
 # CONFIG_RAID_ATTRS is not set
diff -urpN linux-2.6/arch/s390/Kconfig linux-2.6-patched/arch/s390/Kconfig
--- linux-2.6/arch/s390/Kconfig	2006-03-20 06:53:29.000000000 +0100
+++ linux-2.6-patched/arch/s390/Kconfig	2006-03-22 11:46:46.000000000 +0100
@@ -460,6 +460,8 @@ config PCMCIA
 
 source "drivers/base/Kconfig"
 
+source "drivers/connector/Kconfig"
+
 source "drivers/scsi/Kconfig"
 
 source "drivers/s390/Kconfig"
